### DISCUSSION TITLE
fireactjs.com and fireact.dev

### DISCUSSION THREAD

**Discussion by wesley974** [fireactjs/core#1] [Q&A]
Hello,

Are they same project?
Do you plan to add stripe, invoicing?
Possible to choose between one-time/subscription payment?
An home page is missing?

Thanks!

**Comment by chaoming** [comment]
Great questions.

Fireactjs is the replacement project for Fireact.Dev. The key difference is that Fireactjs is shipped as components so it is more developer friendly. You can easily replace or extend your application based on Fireactjs without reading the source code.

My plan is to add the Stripe subscription features as an extension project under @fireactjs. You will need to install an additional NPM package for the subscription features. When that is done, the Fireact.Dev will be migrated to the new @fireactjs/saas.

I don't have a plan for one-time payments yet. My interest is in the SaaS business model. However, it might be just some small adjustment to change it to one-time payments because the subscription payments are managed by Stripe. So it is really depending on the `price_id` in your Stripe product. I will need to look into the one-time payments to give a more solid answer on this, tbh.

You will need to develop the homepage based on what you want to do with the framework for your application. There is an example of the homepage in the documentation. It shows the user information. You can find the example code here: https://fireactjs.com/docs/core-package/development/

Hope the above information helps.

**Reply by wesley974** [reply]
After Stripe integration, user will be able to download his invoice (one-time/each month depending the plan used)?

**Reply by chaoming** [reply]
Yes, the current Fireact.Dev project is able to do that too. There are links to the Stripe invoices.

**Reply by wesley974** [reply]
I will wait your changes in FiReactJs  :)

**Reply by chaoming** [reply]
l am curious about what project you are thinking to build if you don't mind sharing.

**Reply by chaoming** [reply]
@wesley974 The integration with Stripe is completed. Please check out the SaaS components project here: https://github.com/fireactjs/saas
